cmd/cleanenv/main.go

```diff
@@ -0,0 +1,90 @@
+package main
```

Comment:
Maybe a package level comment as to why this tool is written might help for readers wondering what is this for.

Reply:
Good idea. Added in 0563dc6